cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -434,54 +434,72 @@ def _deserialize_gate_op(
                 exponent=arg_func_langs.float_arg_from_proto(
                     operation_proto.xpowgate.exponent,
                     arg_function_language=arg_function_language,
-                    required_arg_name=None,
+                    required_arg_name='exponent',
```

Comment:
What's the point of this change? Seems like this will change functionality by requiring `exponent` to be set.

Reply:
I think I did this for better error messages, but I just reverted it since it may have unintended effects.